SSA/Experimental/IntrinsicAsymptotics.lean

```diff
@@ -601,19 +571,19 @@ theorem subset_entries_matchVar_matchArg
     ma.entries ⊆ varMap.entries :=
   subset_entries_matchVar_matchArg_aux (fun _ _ _ _ _ => subset_entries_matchVar)
 
-instance (t : Ty) : Inhabited t.toType := by
-  cases t <;> dsimp [Ty.toType] <;> infer_instance
+-- TODO: this assumption is too strong, we also want to be able to model non-inhabited types
```

Comment:
really, why do we want to be able to model non-inhabited types? What's the use-case for that?